tests/src/test/scala/org/http4s/multipart/MultipartParserSpec.scala

```diff
@@ -114,8 +145,9 @@ object MultipartParserSpec extends Specification {
       bv.decodeAscii mustEqual Right(expected)
     }
 
-    "produce the body from a two-part input" in {
-      val unprocessedInput = """--_5PHqf8_Pl1FCzBuT5o_mVZg36k67UYI
+    "fail if the header is too large" in {
```

Comment:
Should probably annotate that you have introduced a 100 Byte Cap Here, so that someone who follows doesn't get confused why the same input is supposed to fail in 1 test, and pass another.